src/libm/sleefsimddp.c

```diff
@@ -389,6 +393,13 @@ static INLINE CONST ddi_t rempi(vdouble a) {
   return ret;
 }
 
+#if !defined(DETERMINISTIC)
```

Comment:
Given that the function is always called "xsin" in C code, I would prefer to have the `#if !defined(DETERMINISTIC)` switch _inside_ the function definition:


```
EXPORT CONST vdouble xsin(vdouble d) {
#if !defined(DETERMINISTIC)
// original version
#else // #if !defined(DETERMINISTIC)
// deterministic version
#endif // #if !defined(DETERMINISTIC)
}
```

Could you please apply this change to all the function definitions of this change-set in which you have introduced DETERMINISTIC behavior?

This is quite important because our downstream version of SLEEF is using the attribute `aarch64_vector_pcs` [1],  and having only one declaration instead of two will make merging easier.

[1] https://developer.arm.com/products/software-development-tools/hpc/arm-compiler-for-hpc/vector-function-abi section 2.1